pkg/commands/client/wire_gen.go

```diff
@@ -26,7 +26,7 @@ import (
 func initializeDockerScanner(ctx context.Context, imageName string, artifactCache cache.ArtifactCache, customHeaders client.CustomHeaders, url client.RemoteURL, timeout time.Duration, artifactOption artifact.Option, configScannerOption config.ScannerOption) (scanner.Scanner, func(), error) {
 	scannerScanner := client.NewProtobufClient(url)
 	clientScanner := client.NewScanner(customHeaders, scannerScanner)
-	dockerOption, err := types.GetDockerOption(timeout)
+	dockerOption, err := types.GetDockerOption(timeout, artifactOption.InsecureSkipTLS)
```

Comment:
Did you edit this file directly? If you used `make wire`, it is ok. If it is not the case, you should not edit this file since it is generated.